zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -1473,29 +1241,14 @@ impl<'a, F: Field> EVMConstraintBuilder<'a, F> {
         self.reversion_info(call_id, false)
     }
 
-    pub(crate) fn reversion_info_write(
+    pub(crate) fn reversion_info_write_unchecked(
```

Comment:
Discussion:  I noticed in [begin_tx.rs](https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/evm_circuit/execution/begin_tx.rs#L81) and [ReversionInfo](https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/evm_circuit/util/constraint_builder.rs#L96-L104) main branch there is no range check, does it safe?